docs/reference/modules/node.asciidoc

```diff
@@ -156,6 +156,12 @@ discovery.zen.minimum_master_nodes: 2 <1>
 ----------------------------
 <1> Defaults to `1`.
 
+Critical clusters must have at least three master-eligible nodes, with
+`minimum_master_nodes` set accordingly, in order to remain available even when
+a node fails. A <<rolling-upgrades,rolling upgrade>> also requires at least
```

Comment:
can you add "which don't allow any downtime" so it's clear what we're aiming at?